src/main/java/org/folio/service/orders/PurchaseOrderLineService.java

```diff
@@ -193,65 +173,6 @@ public Future<List<PoLine>> getPoLinesByOrderId(String orderId, RequestContext r
     return promise.future();
   }
 
-  public Future<List<CompositePoLine>> getCompositePoLinesByOrderId(String orderId, RequestContext requestContext) {
-    return getOrderLines("purchaseOrderId==" + orderId, 0, Integer.MAX_VALUE, requestContext)
-      .compose(poLines -> requestContext.getContext().owner()
-        .<List<Future<CompositePoLine>>>executeBlocking(event -> {
-          if (CollectionUtils.isEmpty(poLines)) {
-            event.complete(new ArrayList<>());
-            return;
-          }
-
-          Semaphore semaphore = new Semaphore(SEMAPHORE_MAX_ACTIVE_THREADS, requestContext.getContext().owner());
```

Comment:
good that we are going to delete this complicated and unused code that slows down the system and adds unnesessary conversions and compexity